src/chapters/comm_brevity.tex

```diff
@@ -0,0 +1,260 @@
+\chapter{COMM BREVITY}
+\thumbtab{Comm Brevity}{9}
+\localtableofcontents
+\thispagestyle{plain}
+\cleardoublepage
+
+\section{A-A COMMUNICATION}
+
+\subsection{INTERCEPT}
+
+\begin{tcoloritemize}
+    \blueitem[ANGELS]
+    Height of FRIENDLY aircraft in thousands of feet from MSL.
+
+    \medskip
+    \textbf{Example}
+    \begin{center}
+        \begin{minipage}{0.9\textwidth}
+            SIMBA 1, AT ANGELS 20.
+        \end{minipage}
+    \end{center}    
+
+    \blueitem[AZIMUTH]
+    1. A picture label describing the lateral separation of two GROUPS.\newline
+    2. Direction to a threat.
+
+    \medskip
+    \textbf{Example}
```

Comment:
Is this example for 1. or for 2.? 
I would use a `\ref{...}` here to automatically match enumeration and formatting of list above.
suggested label: `\label{subsec:comm_brevity:aa_comm:intercept:azimuth:<picture or direction>}` for the two options and matching `\ref{...}`